File: add/codesnippet/CSharp/f-system.uri.schemedelim_1.cs
            string address = "www.contoso.com";
            string uriString = String.Format("{0}{1}{2}/", Uri.UriSchemeHttp, Uri.SchemeDelimiter, address); 
 #if OLDMETHOD           
            Uri result;
            if (Uri.TryParse(uriString, false, false, out result) == true)
                Console.WriteLine("{0} is a valid Uri", result.ToString());
            else
                Console.WriteLine("Uri not created");
#endif
            Uri result = new Uri(uriString);
            if (result.IsWellFormedOriginalString())
                Console.WriteLine("{0} is a well formed Uri", uriString);
            else
                Console.WriteLine("{0} is not a well formed Uri", uriString);

File: add/codesnippet/CPP/f-system.uri.schemedelim_1.cpp
         String^ address = "www.contoso.com";
         String^ uriString = String::Format( "{0}{1}{2}",
            Uri::UriSchemeHttp, Uri::SchemeDelimiter, address );

#if OLDMETHOD           
         Uri^ result;
         if ( Uri::TryParse( uriString, false, false, result ) == true )
         {
            Console::WriteLine( "{0} is a valid Uri", result );
         }
         else
         {
            Console::WriteLine( "Uri not created" );
         }
#endif
         Uri ^result = gcnew Uri(uriString);
         if (result->IsWellFormedOriginalString())
             Console::WriteLine("{0} is a well formed Uri", uriString);
         else
             Console::WriteLine("{0} is not a well formed Uri", uriString);